nock/chat/messages/slack/GetMessages/old/recording-17e477fe899e29fb93a2117a6a9baadf_0.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
Old recordings will be deleted? Or what is their purpose?

Reply:
This is still not solved, I thought we could discuss it more as research about integration lifecycle will be done.

Reply:
I removed them for now, they are now as a default ones after revert.